diff -r -U3 Middlewares/Third_Party/FreeRTOS/Source/include/StackMacros.h Middlewares_b/Third_Party/FreeRTOS/Source/include/StackMacros.h
--- a/Middlewares/Third_Party/FreeRTOS/Source/include/StackMacros.h	2018-04-30 18:55:03.000000000 +0200
+++ b/Middlewares/Third_Party/FreeRTOS/Source/include/StackMacros.h	2018-04-30 19:08:26.000000000 +0200
@@ -94,7 +94,7 @@
 		/* Is the currently saved stack pointer within the stack limit? */								\
 		if( pxCurrentTCB->pxTopOfStack <= pxCurrentTCB->pxStack )										\
 		{																								\
-			vApplicationStackOverflowHook( ( TaskHandle_t ) pxCurrentTCB, pxCurrentTCB->pcTaskName );	\
+			vApplicationStackOverflowHook( ( TaskHandle_t ) pxCurrentTCB, (signed char *) pxCurrentTCB->pcTaskName );	\
 		}																								\
 	}
 
diff -r -U3 Middlewares/Third_Party/FreeRTOS/Source/tasks.c Middlewares_b/Third_Party/FreeRTOS/Source/tasks.c
--- a/Middlewares/Third_Party/FreeRTOS/Source/tasks.c	2018-04-30 18:55:03.000000000 +0200
+++ b/Middlewares/Third_Party/FreeRTOS/Source/tasks.c	2018-04-30 19:09:52.000000000 +0200
@@ -427,7 +427,7 @@
 
 /* Callback function prototypes. --------------------------*/
 #if(  configCHECK_FOR_STACK_OVERFLOW > 0 )
-	extern void vApplicationStackOverflowHook( TaskHandle_t xTask, char *pcTaskName );
+	extern void vApplicationStackOverflowHook( TaskHandle_t xTask, signed char *pcTaskName );
 #endif
 
 #if( configUSE_TICK_HOOK > 0 )
